src/main/java/com/booksaw/betterTeams/customEvents/post/PostPurgeEvent.java

```diff
@@ -0,0 +1,44 @@
+package com.booksaw.betterTeams.customEvents.post;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This event is called after a team purge has been completed.
+ * This event cannot be cancelled since it occurs after the purge.
+ *
+ * To modify or cancel the purge, use {@link PurgeEvent}.
+ *
+ * @author svaningelgem
+ */
+public class PostPurgeEvent extends Event implements Cancellable {
```

Comment:
if we do not want the event to be cancellable, why are we implementing the Cancellable interface and throwing errors, can we not just not implement the interface?

Reply:
fixed.